### ISSUE TITLE
[Feature Request] Custom Contents in Sidebar for ExeryPage

### ISSUE THREAD

**Issue by ramesh-dada** [vuepress/vuepress-next#343]
### Discussed in https://github.com/vuepress/vuepress-next/discussions/340

<div type='discussions-op-text'>

<sup>Originally posted by **ramesh-dada** August  3, 2021</sup>
</div>

### Description
I would like to have custom links and different sidebar for every single page I don’t want my headings render as table of contents in sidebar on some pages a option with yaml to enable or disable also I would like to have custom links that are different on different pages that maybe external or internal as well content like this.

![](https://user-images.githubusercontent.com/75124237/127965901-b50de16c-3d65-4606-a998-64de7c86db5c.png)

### Proposed Solution

We can allow modifications with md file like we can allow users to add list with links with markdown formatting.
In short need to customize content of sidebar but different content on different pages.

**Comment by meteorlxy** [comment]
Please read docs carefully before creating an issue.

https://v2.vuepress.vuejs.org/reference/default-theme/frontmatter.html#sidebar
